### ISSUE TITLE
Too many issues

### ISSUE THREAD

**Issue by mifi** [kelseyhightower/nocode#3904]
I think this project has too many issues for such little code. I’m sorry that I contributed with yet another issue

**Comment by Optimissedit** [comment]
Agreed. Most of these need to get knocked off so we can focus on genuine issues regarding this project.
